make/autoconf/jdk-options.m4

```diff
@@ -472,6 +472,24 @@ AC_DEFUN_ONCE([JDKOPT_SETUP_STATIC_BUILD],
   AC_SUBST(STATIC_BUILD)
 ])
 
+################################################################################
+#
+# jmod options.
+#
+AC_DEFUN_ONCE([JDKOPT_SETUP_JMOD_OPTIONS],
+[
+  AC_ARG_WITH(jmod-compress, [AS_HELP_STRING([--with-jmod-compress],
```

Comment:
I would recommend using UTIL_ARG_WITH here instead. I would also prefer if the default was specified in configure (using the DEFAULT argument to UTIL_ARG_WITH) so that it's visible in the help output and printed in the configure log for user transparency.

Reply:
Changed!